src/main/java/thaumicboots/api/serverfiles/PacketSpeedToggle.java

```diff
@@ -0,0 +1,40 @@
+package thaumicboots.api.serverfiles;
+
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.ItemStack;
+
+import cpw.mods.fml.common.network.simpleimpl.IMessage;
+import cpw.mods.fml.common.network.simpleimpl.IMessageHandler;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import io.netty.buffer.ByteBuf;
+import thaumicboots.api.ItemBoots;
+
+public class PacketSpeedToggle implements IMessage, IMessageHandler<PacketSpeedToggle, IMessage> {
+
+    @Override
+    public void fromBytes(ByteBuf byteBuf) {
+        // not needed
+    }
+
+    @Override
+    public void toBytes(ByteBuf byteBuf) {
+        // not needed
+    }
+
+    @Override
+    @SideOnly(Side.CLIENT)
```

Comment:
`@SideOnly` is not needed here

Reply:
trust me, ***it does***

otherwise it'll crash the server.